src/main/kotlin/no/skatteetaten/aurora/boober/service/internal/MountCreator.kt

```diff
@@ -72,3 +74,21 @@ private fun createDatabaseMounts(deploymentSpec: AuroraDeploymentSpec,
 
     return databaseMounts
 }
+
+private fun createToxiProxyMounts(deploymentSpec: AuroraDeploymentSpec): List<Mount> {
+    val sidecar = "toxiproxy"
+
+    return deploymentSpec.deploy?.toxiProxy?.let {
+        listOf(Mount(
+            path = "/u01/config",
+            type = MountType.ConfigMap,
+            mountName = "${sidecar}-volume",
+            volumeName = "${sidecar}-config",
+            exist = false,
+            content = mapOf("config.json" to getToxiProxyConfig()),
+            targetContainer = sidecar))
```

Comment:
Will it ever happen that a mount should be mounted up in several containers? Should this be a set and we should check contains and not equals?

Reply:
We need to discuss this change. I am not sure you even know in all cases where something should be mounted when you create it. What if this logic resides in the function that handles toxiproxy in the DC? You look for a mount with a given name or something like it?